From 0f8568df371cf808c9ba3d22b10ef7c719b93d5e Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 29 Apr 2021 08:08:56 -0500
Subject: [PATCH 80/83] dt-bindings: imx8mn: Fix reference to IMX8MN_CLK_UART1

The entries in the dt-bindings that describe the various clocks
need to be unique.  That wasn't the case for IMX8MN_CLK_UART1.
It was using a duplicated reference which meant that attempts to
re-parent this clock were being referenced by another.
Fix this by making the clock reference unique.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 include/dt-bindings/clock/imx8mn-clock.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/dt-bindings/clock/imx8mn-clock.h b/include/dt-bindings/clock/imx8mn-clock.h
index f1bd535c0883..f4a18541f987 100644
--- a/include/dt-bindings/clock/imx8mn-clock.h
+++ b/include/dt-bindings/clock/imx8mn-clock.h
@@ -125,7 +125,7 @@
 #define IMX8MN_CLK_I2C2				106
 #define IMX8MN_CLK_I2C3				107
 #define IMX8MN_CLK_I2C4				118
-#define IMX8MN_CLK_UART1			119
+#define IMX8MN_CLK_UART1			109
 #define IMX8MN_CLK_UART2			110
 #define IMX8MN_CLK_UART3			111
 #define IMX8MN_CLK_UART4			112
-- 
2.17.1

